Sources/Primitives/UnfairLock.swift

```diff
@@ -27,7 +27,7 @@ private typealias NativeLock = pthread_mutex_t
 ///
 /// The lock is `~Copyable` to prevent accidental duplication which
 /// could lead to synchronization issues.
-public struct UnfairLock: ~Copyable, Sendable {
+public final class UnfairLock: @unchecked Sendable {
```

Comment:
This is rather unfortunate - perhaps we should introduce an internal wrapper on Linux that wraps the `pthread_t` into a class?